web/models/employee.ex

```diff
@@ -61,4 +68,11 @@ defmodule Appointments.Employee do
     |> DB.add_reset_token(key)
   end
 
+  @doc """
+  Returns the full name of the employee
+  """
+  def full_name(employee) do
+    "#{employee.first_name} #{employee.last_name}"
```

Comment:
Haha AFAICT virtual fields are fields that can be stored and manipulated in changesets but _are not_ persisted to the database.